apex/rsa-sign-service.cls

```diff
@@ -0,0 +1,103 @@
+public class RsaSignService {
+
+	private static final String AUTHORIZATION_SCHEME = 'CS-Elastic';
+	private static final Integer NONCE_LENGTH = 5;
+	private static final String NONCE_CHARACTERS = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789abcdefghijklmnopqrstuvwxyz';
+	private static final Integer NONCE_CHARACTERS_LENGTH = NONCE_CHARACTERS.length();
+
+	private HttpRequest getHeadersRSA(HttpRequest request, String payload) {
+		request.setHeader('Organization', ORG_ID);
+		request.setHeader('Authorization', getAuthorizationHeader(
```

Comment:
There is no method getAuthorizationHeader, you probably meant to use generateRsaAuthorizationHeader. Suggest to rename generateRsaAuthorizationHeader to getAuthorizationHeader.

Reply:
I see this is changed in next commit. Please ignore.